client/js/shout.js

```diff
@@ -382,6 +382,19 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("highlights") !== null) {
+		$("#highlights").val($.cookie("highlights")).change();
+	}
+
+	var highlights = [];
+
+	$("#highlights").on("change", function() {
+		var self = $(this);
+		var string = self.val();
```

Comment:
Ah yes, good call. I was copying it from the PR of the theme selector, and it does more with the element, and just didn't notice I can combine that.